.github/workflows/test-integration-20.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       - name: Setup
         uses: ./.github/actions/setup
         with:
-          node-version: 18
+          node-version: 20
```

Comment:
Do other values in this file need to be updated as well? There still seem to be a couple of references to Node 18.

Reply:
Good catch in both cases :+1: